src/openrct2/platform/Crash.cpp

```diff
@@ -168,6 +168,15 @@ static bool OnCrash(
     try
     {
         auto exporter = std::make_unique<S6Exporter>();
+
+        // Make sure the save is using the current viewport settings.
+        viewport_set_saved_view();
+
+        // Export all loaded objects to avoid having custom objects missing in the reports.
+        auto context = GetContext();
```

Comment:
This shadows function argument, but as the argument is not used I suggest commenting it out -> `void* /*context*/` in line 90